examples/poller.py

```diff
@@ -122,27 +127,25 @@ def receivedSlot(self, channelName: str, content: Any):
             if isinstance(content, dict):
                 self.updateDB(content)
             else:
-                print("The message for the channel db should be a dictionary.")
+                logger.error("The message for the channel db should be a dictionary.")
         else:
-            print(f"The message was ignored because "
-                  f"the treatment for the channel {channelName} is not implemented.")
+            logger.error("The message was ignored because "
+                        "the treatment for the channel %s is not implemented.", channelName)
 
     @pyqtSlot()
     def setPeriod(self):
         """Sets the polling period."""
         period = self.viewerFrame.periodBox.value()
         self.timer.start(1000 * period)
-        self.broadcast("log", f"Period is set as {period}s.")
+        logger.info("Period is set as %ds.", period)
 
     @pyqtSlot()
     def setDB(self):
         """Sets the database to store the polled number."""
         self.dbName = self.viewerFrame.dbBox.currentText()
-        self.broadcast(
-            "log", 
-            f"Polled database is set as {self.dbName}." if self.dbName
-            else "Polled database is not selected."
-        )
+        if self.dbName:
+            logger.info("Database to store is set as %s.", self.dbName)
+        else: logger.warning("Database to store is not selected.")
```

Comment:
New line!

Also, in `numgen.py`, you handled this case with `logger.info()`.
Please unify both statements of two files.